### ISSUE TITLE
Unnecessary error when using css helper function

### ISSUE THREAD

**Issue by protoEvangelion** [microsoft/typescript-styled-plugin#81]
This is related to https://github.com/Microsoft/typescript-styled-plugin/issues/74

When using the css helper function inline on a component prop, `typescript-styled-plugin` throws 2 unnecessary errors:

```js
<Component animation={css`${condition ? animation1 : animation2} 1s linear`} />
```
![image](https://user-images.githubusercontent.com/20076677/48307460-d5d47a00-e501-11e8-87ef-4ca5f30c3deb.png)

![image](https://user-images.githubusercontent.com/20076677/48307463-e258d280-e501-11e8-9dff-52dd60df374f.png)

**Comment by mjbvz** [comment]
Is that a valid styled component definition?

**Comment by protoEvangelion** [comment]
The context of this is interpolating keyframes. In styled-components v4 they require us to use the `css` [helper](https://www.styled-components.com/docs/api#helpers) to do this:

![image](https://user-images.githubusercontent.com/20076677/48445678-fa239700-e74b-11e8-9ecd-b0e7331b8701.png)

In my case I am trying to create a reusable component that allows you to pass in a custom animation.

**Comment by protoEvangelion** [comment]
I was able to disable the error by omitting `css` from tags in compilerOptions:

```json
{
  "compilerOptions": {
    "plugins": [
      {
        "name": "typescript-styled-plugin",
        "tags": ["styled"]
      }
    ]
  }
}
```

Does this look right?

**Comment by EricVanDerDijs** [comment]
I'm having the same issue. I'm wraping up a functional component to conditionally add styles depending on props. Working on Ubuntu 18.04
![ts-styled-plugin-error](https://user-images.githubusercontent.com/29189893/48579694-98059600-e8f3-11e8-9c1e-277a3792830b.png)

**Comment by protoEvangelion** [comment]
@EricVanDerDijs I placed this in `tsconfig.json` under the `compilerOptions` key. Try that.

**Comment by EricVanDerDijs** [comment]
Thanks, it did help. At first I tried with the jsconfig.json file.

**Comment by protoEvangelion** [comment]
Are you adding your styles inline on an html tag using the `css` helper?

```jsx
<Component prop={css`...`} />
```

If not, this might be a different issue.

**Comment by EricVanDerDijs** [comment]
No, i'm using a simple style(Component)`  /*  STYLES   */  `; function, but have a similar error, and recently others have reported a similar error message ("expected }" on files that don't have syntax error) like #83 (others that seems alike are #82 #51 ). should I start a new issue?

**Comment by karolisgrinkevicius** [comment]
I'm also getting this kind of error even when using `"tags": ["styled"]` in tsconfig.

**Comment by mjbvz** [comment]
Closing as this package has been deprecated in favor of the [official styled-components/typescript-styled-plugin fork]( https://github.com/styled-components/typescript-styled-plugin)

After updating to that fork, follow up in the styled-components repo if this is still an issue
